client/landing/gutenboarding/onboarding-block/acquire-intent/vertical-select/index.tsx

```diff
@@ -59,109 +57,129 @@ const VerticalSelect: React.FunctionComponent< Props > = ( { onSubmit } ) => {
 	const { siteVertical, siteTitle } = useSelect( select => select( ONBOARD_STORE ).getState() );
 	const { setSiteVertical, resetSiteVertical } = useDispatch( ONBOARD_STORE );
 
-	const [ inputValue, setInputValue ] = React.useState( siteVertical?.label ?? '' );
+	const inputText = inputRef?.current?.innerText || '';
+	const isInputEmpty = ! inputText.length;
+	const showArrow = ! siteTitle && ! siteVertical && inputText.length > 2;
 
 	const animatedPlaceholder = useTyper(
 		[ NO__( 'football' ), NO__( 'shopping' ), NO__( 'cars' ), NO__( 'design' ), NO__( 'travel' ) ],
-		inputValue.length === 0
+		isInputEmpty
 	);
 
-	const normalizedInputValue = inputValue.trim().toLowerCase();
-	const hasValue = !! normalizedInputValue.length;
+	const updateSuggestions = ( inputValue: string ) => {
+		if ( inputValue.length < 3 ) {
+			setSuggestions( [] );
+			return;
+		}
 
-	let suggestions: Suggestion[];
+		const normalizedInputValue = inputValue.toLowerCase();
 
-	if ( ! normalizedInputValue ) {
-		suggestions = [];
-		resetSiteVertical();
-	} else {
-		suggestions = verticals.filter( vertical =>
+		let newSuggestions = verticals.filter( vertical =>
 			vertical.label.toLowerCase().includes( normalizedInputValue )
 		);
 
 		// Does the verticals list include an exact match? If it doesn't, we prepend the user-suppied
 		// vertical to the list.
 		if (
-			! suggestions.some( suggestion => suggestion.label.toLowerCase() === normalizedInputValue )
+			! newSuggestions.some( suggestion => suggestion.label.toLowerCase() === normalizedInputValue )
 		) {
 			// User-supplied verticals don't have IDs.
-			suggestions.unshift( { label: inputValue.trim() } );
+			newSuggestions.unshift( { label: inputValue.trim(), id: '', slug: '' } );
 		}
 
 		// If there is only one suggestion and that suggestion matches the user input value,
 		// do not show any suggestions.
 		if (
-			suggestions.length === 1 &&
-			suggestions[ 0 ].label.toLowerCase() === normalizedInputValue
+			newSuggestions.length === 1 &&
+			newSuggestions[ 0 ].label.toLowerCase() === normalizedInputValue
```

Comment:
We'll have the entire UI localised including vertical suggestions (here is some spike we made in this direction: https://github.com/Automattic/wp-calypso/pull/37756). But I can see your point since a lot of users are using international keyboards (English US I guess) to write words in their own language without using diacritics. Adding a Todo comment ✍️